### ISSUE TITLE
VSCode always asks for API key

### ISSUE THREAD

**Issue by 0xcaffebabe** [wakatime/vscode-wakatime#324]
Even regenerate new API key input, wakatime plugin still asks the API key.

No error log or warning log.

![image](https://user-images.githubusercontent.com/34188127/215256221-61db9791-c9b7-4245-afee-9cac35dcc0c2.png)

**Comment by alanhamlett** [comment]
If you enable debug mode what logs do you see in your `~/.wakatime.log` file?

Here's how to enable debug mode: https://github.com/wakatime/vscode-wakatime#troubleshooting

**Comment by 0xcaffebabe** [comment]
> If you enable debug mode what logs do you see in your `~/.wakatime.log` file?
> 
> Here's how to enable debug mode: https://github.com/wakatime/vscode-wakatime#troubleshooting

I see these warning:

![image](https://user-images.githubusercontent.com/34188127/215257022-f855dcfe-ba4a-4450-afd4-8615df0f9ad4.png)

is network problem?

**Comment by alanhamlett** [comment]
Yes, that means the plugin isn't able to download/update wakatime-cli from GitHub. Do you need a proxy to connect to the internet?

**Comment by 0xcaffebabe** [comment]
> Yes, that means the plugin isn't able to download/update wakatime-cli from GitHub. Do you need a proxy to connect to the internet?

I set the proxy in vscode, but it still cant download, has any way to manual to download 'wakatime-cli-windows-amd64' file and replace local file?

![image](https://user-images.githubusercontent.com/34188127/220223060-61b3a9c9-0db1-4ec7-b6fa-34435d4872ef.png)

**Comment by alanhamlett** [comment]
> has any way to manual to download 'wakatime-cli-windows-amd64' file and replace local file?

Yes, download [this zip file](https://github.com/wakatime/wakatime-cli/releases/download/v1.67.0/wakatime-cli-windows-amd64.zip) then extract it and copy `wakatime-cli-windows-amd64.exe` to `$HOME/.wakatime/wakatime-cli-windows-amd64.exe` and `$HOME/.wakatime/wakatime-cli.exe`.

You can also try running [this install script](https://github.com/wakatime/vim-wakatime/blob/master/scripts/install_cli.py) if you're not able to manually download and extract the zip.

**Comment by alanhamlett** [comment]
Let's discuss further in #362.
